contrib/systemd/spacenavd.service

```diff
@@ -1,12 +1,10 @@
 [Unit]
 Description=3Dconnexion Input Devices Userspace Driver
-After=syslog.target
 
 [Service]
-Type=forking
+Type=exec
```

Comment:
Systemd doesn't need you to fork for it. That's what it is good at, containerizing and forking.